PWGLF/Tasks/Nuspex/spectraTOF.cxx

```
@@ -18,6 +18,7 @@
 ///
 
 // O2 includes
+// o2-linter: disable=all
```

First, there is no such directive, second, you are not supposed to silence error reports instead of fixing the reported errors.

---

Thank I will keep in mind next time.